assets/css/main.css

```diff
@@ -0,0 +1,342 @@
+@import 'reset.css';
+
+
+nav{
+  border-bottom: 1px solid lightgray;
+  height: 50px;
+  position: fixed;
+  background: #fff;
+  width: 100%;
+}
+
+.mobile-menu{
+  display:  none;
+}
+
+nav li{
+  display: inline;
+  list-style: none;
+  padding: 40px;
+  text-transform: uppercase;
+  line-height: 3;
+  text-align: center;
+}
+
+a{
+  text-decoration: none;
+  color: #000;
+}
+
+a:hover{
+  color: red;
+}
+
+nav a{
+  text-decoration: none;
+  color: #000;
+}
+nav a:hover {
+  color: red;
+}
+
+.hero{
+  display: grid;
+}
+
+.col-6{
+  grid-column: span 6;
+  text-align: center;
+}
+
+.col-6 li{
+  color: red;
+  margin: 20px;
+}
+
+.col-6 p {
+  color: red;
+}
+h1{
+  line-height: 1;
+  color: red;
+  font-size: 55px;
+  font-weight: 10;
```

Comment:
This value is an invalid `font-weight` value.

[Here](https://htmldog.com/references/css/properties/font-weight/) is a list of all valid `font-weight` values.